packages/browser-agent/README.md

```diff
@@ -1,27 +1,148 @@
-This is the main module that clients would import into their applications.
+[![New Relic Experimental header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Experimental.png)](https://opensource.newrelic.com/oss-category/#new-relic-experimental)
 
-It provides APIs to initialize the agent:
+# New Relic Browser Agent
 
+The New Relic Browser Agent instruments your website and provides observability into the performance and behavior of your application. This NPM Library is an ***in-progress*** implementation of the New Relic Browser Agent, accessible through `NPM`. Please see the [Differences](#differences) and [Features](#features) sections to compare how this library differs from the other offerings of the New Relic Browser Agent.
+
+## Installation
+
+### Using Package Managers
+
+Using [npm](https://npmjs.org):
+
+```bash
+npm install @newrelic/browser-agent
+```
+
+or [yarn](https://yarnpkg.com/)
+
+```bash
+yarn add @newrelic/browser-agent
+```
+
+### Directly in HTML/JS
+
+Using [unpkg](https://unpkg.com/)
+
+```html
+<script src="https://unpkg.com/@newrelic/browser-agent/bundled"></script>
 ```
-import nr from 'nr-browser-core'
 
-nr.setConfiguration({
-  jserrors: {
-    harvestTimeSeconds: 30
-  }
+## Usage Examples
+
+### Basic Setup
+
+```javascript
+<< App.js >>
+// initialize the agent as close to 
+// the top level of the application as possible
+import NR from '@newrelic/browser-agent'
+const options = {
+    // See 'Configuring your application'
+}
+NR.start(options).then(() => {
```

Comment:
This pattern has changed some in recent commits.  it is now requires an instance to be made before starting

```javascript
const nr = new NR()
nr.start(options).then(...)
```

Reply:
also, of note (FYI), along with the default export as shown above, a user can also do a named import

```javascript
import { BrowserAgent } from '@newrelic/browser-agent'
```